mockgen/mockgen.go

```diff
@@ -304,10 +304,16 @@ func (g *generator) Generate(pkg *model.Package, outputPkgName string, outputPac
 	}
 	sort.Strings(sortedPaths)
 
+	importPaths := make([]string, 0)
```

Comment:
No need to make the slice as you can append to a nil slice
```suggestion
var importPaths []string
```

Reply:
Sure